applications/serial_lte_modem/src/lwm2m_carrier/slm_at_carrier.c

```diff
@@ -8,11 +8,14 @@
 #include <time.h>
 #include <lwm2m_carrier.h>
 #include <zephyr/logging/log.h>
+#include <modem/nrf_modem_lib.h>
 #include "slm_at_host.h"
 #include "slm_util.h"
 
 LOG_MODULE_REGISTER(slm_carrier, CONFIG_SLM_LOG_LEVEL);
 
+#define REBOOTLESS_FOTA 0
```

Comment:
Should this be changed to a Kconfig item for customers to be able to build with NIDD support?

Reply:
This is TEMP code for rebootless MFW FOTA debug